ultraLatency.py

```diff
@@ -228,15 +225,16 @@
             sys.stdout.flush()
             pass
         for VC in range(LATENCY_MAX-LATENCY_MIN+1):
-            gemData.latency[0]   = int((dataNow[VC] & 0xff000000) >> 24)
-            gemData.Nhits[0] = int(dataNow[VC] & 0xffffff)
+            latency   = int((dataNow[VC] & 0xff000000) >> 24)
+            Nhits = int(dataNow[VC] & 0xffffff)
+            gemData.setScanResults(latency, Nhits)
```

Comment:
OK, more what I was trying to say was, the `fill`  you have at line 234 could be something akin to:
`fill(vt1=vt1val,vthr=vthrval,lat=latval)`
and in the class you'd define it:
```python
def fill(**kwargs):
    if "vt1" in kwargs:
      self.vt1[0] = kwargs["vt1"]
...
```
This would allow the call to encapsulate all the data, the `dataTree` class would keep everything organized, and it would simplify the `ultraX` code

Reply:
Implemented, yes this was a better approach.  Thanks for suggestion.